dev/CommandBarFlyout/CommandBarFlyoutCommandBar.h

```diff
@@ -66,6 +66,7 @@ class CommandBarFlyoutCommandBar :
     tracker_ref<winrt::FrameworkElement> m_primaryItemsRoot{ this };
     tracker_ref<winrt::FrameworkElement> m_secondaryItemsRoot{ this };
     tracker_ref<winrt::ButtonBase> m_moreButton{ this };
+    tracker_ref<winrt::UIElement> m_overflowPopup{ this };
```

Comment:
Correct me if I'm wrong, but I don't think that this is used either 😅

Reply:
you're right! fixed